sound/soc/sof/loader.c

```diff
@@ -65,6 +90,9 @@ int snd_sof_fw_parse_ext_data(struct snd_sof_dev *sdev, u32 bar, u32 offset)
 		case SOF_IPC_EXT_WINDOW:
 			ret = get_ext_windows(sdev, ext_hdr);
 			break;
+		case SOF_IPC_EXT_CC_INFO:
+			ret = get_cc_info(sdev, ext_hdr);
+			break;
```

Comment:
shouldn't we have a check that this is only provided once? Otherwise you'll have a warning with the debugfs item created multiple times.

Reply:
ok